collections/_article/rendering-priorities-january-2024.md

```diff
@@ -0,0 +1,116 @@
+
+**3. Acyclic render graph optimization ([GH-84976](https://github.com/godotengine/godot/pull/84976)) (merged for 4.3)**
+
+This is both an optimization and a feature. The Acyclic Render Graph (ARG) is a tool to automatically record and re-order rendering commands in the RenderingDevice backend. This is extremely beneficial for 2 reasons: 1) it enables optimizations that are otherwise not possible in a general purpose renderer and 2) it simplifies the RenderingDevice API to make it much easier to use (this benefits us and users who use the RenderingDevice API directly). So far this PR has shown promising performance improvements on a range of devices and especially when many particle systems are in use. 
+
+**4. Rendering Hooks ([GH-80214](https://github.com/godotengine/godot/pull/80214))**
+
+Rendering hooks are our way of allowing users to insert custom RenderingDevice code in between draw passes in the built in renderer. This paves the way for many custom effects that rely on accessing internal resources or that take place during rendering. Particularly, this will open up a channel to create custom post processing effects. The implementation is still quite low-level and requires using the RenderingDevice API directly, but we consider this a first step in allowing users much greater control over rendering.
+
+**5. Metal rendering driver**
+
+Recently Stuart Carnie began work on a Metal backend for Godot. The work is significantly underway and is already showing a lot of promise. A Metal backend will allow us to massively improve user experience on Apple devices (both iOS and MacOS). Including fixing outstanding bugs and improving performance.
+
+**6. SDFGI refactor ([GH-86267](https://github.com/godotengine/godot/pull/86267))**
+
+After initially writing SDFGI, Juan had many ideas to improve it which we discussed at length. He captured his ideas in a handly slideshow which we shared publicly in the Fall of 2022 (https://www.docdroid.net/YNntL0e/godot-sdfgi-pdf). Juan has now begun work on those ideas and they led him to a slightly different approach to GI, using the Heirarchical Digital Differential Analyzer (HDDA) algorithm instead of Signed Distance Fields (SDF). Accordingly, we will be renaming SDFGI to "Dynamic GI" to make the name independent of the algorithm used. The settings will change slightly in this refactor, but both performance and quality will increase. The end goal is to provide settings that will make this run fast enough for use on integrated graphics cards.
```

Comment:
```suggestion
After initially writing SDFGI, Juan had many ideas to improve it which we discussed at length. He captured his ideas in a handy slideshow which we shared publicly in the Fall of 2022 (https://www.docdroid.net/YNntL0e/godot-sdfgi-pdf). Juan has now begun work on those ideas and they led him to a slightly different approach to GI, using the Heirarchical Digital Differential Analyzer (HDDA) algorithm instead of Signed Distance Fields (SDF). Accordingly, we will be renaming SDFGI to "Dynamic GI" to make the name independent of the algorithm used. The settings will change slightly in this refactor, but both performance and quality will increase. The end goal is to provide settings that will make this run fast enough for use on integrated graphics cards.
```
Also "Fall" is hemisphere-centric. Maybe better to say a month?

Reply:
```suggestion
After initially writing SDFGI, Juan had many ideas to improve it which we discussed at length. He captured his ideas in a handly slideshow which we shared publicly in November 2022 (https://www.docdroid.net/YNntL0e/godot-sdfgi-pdf). Juan has now begun work on those ideas and they led him to a slightly different approach to GI, using the Heirarchical Digital Differential Analyzer (HDDA) algorithm instead of Signed Distance Fields (SDF). Accordingly, we will be renaming SDFGI to "Dynamic GI" to make the name independent of the algorithm used. The settings will change slightly in this refactor, but both performance and quality will increase. The end goal is to provide settings that will make this run fast enough for use on integrated graphics cards.
```